README.adoc

```diff
@@ -46,7 +46,7 @@ to directly convert JSON messages into Java objects. The second approach consist
 using the Java API for JSON Processing (JSON-P) to process the JSON.
 
 The REST service that provides the artists and albums resources has already been written
```

Comment:
Global questionfor clarification: are values such as `http://localhost:9080/artists` and `http://localhost:9080/artists/total` endpoints or URLs? These terms are both used but I think it should be one term consistently, unless the examples are fundamentally different, in which case it should be clear how they are different.

My understanding is that something like `/artists/total/<artist>` is an endpoint, while `http://localhost:9080/artists/total/<artist>` is a URL, which includes the endpoint, but also other information for the HTTP address. Is this correct?

Reply:
yes, `http://localhost:9080/artists/total/<artist>` is an URL and `/artists/total/<artist>` is an endpoint in this guide.

If just read from an URL itself, it cannot 100% tell what is the endpoint. e.g. `http://localhost:9080/a/b/c`.  endpoint can be either
- `/a/b/c` if the app context root is `/`
- `/b/c` if the app context root is `/a`
- `/c` if the app context root is `/a/b`

Reply:
ok, if the endpoint is the part of the URL we want to call attention to, can consistently refer to them as endpoints